lib/msal-core/src/telemetry/TelemetryEvent.ts

```diff
@@ -0,0 +1,56 @@
+
+import { v4 as uuid } from "uuid";
+import { TELEMETRY_BLOB_EVENT_NAMES } from "./TelemetryConstants";
+import {
+    EVENT_NAME_PREFIX,
+    EVENT_NAME_KEY,
+    START_TIME_KEY,
+    ELAPSED_TIME_KEY
+} from "./TelemetryConstants";
+
+export default class TelemetryEvent {
+
+    private startTime: number;
+    protected event: any; // TODO TYPE THIS
+    public eventId: string;
+
+    constructor(eventName: string, correlationId: string) {
+
+        this.startTime = new Date().getTime();
+        this.eventId = uuid();
+        this.event = {
+            [`${EVENT_NAME_PREFIX}${EVENT_NAME_KEY}`]: eventName,
+            [`${EVENT_NAME_PREFIX}${START_TIME_KEY}`]: this.startTime,
+            [`${EVENT_NAME_PREFIX}${ELAPSED_TIME_KEY}`]: -1,
```

Comment:
What does `-1` represent here? Is it just used for semantics, or there a programmatic reason?

Reply:
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/master/src/Microsoft.Identity.Client/TelemetryCore/Internal/Events/EventBase.cs#L28

Default used to denote that an elapsed time has not been set on this event, for consistency with other msal libs